src/ui/dialog/dialog.tsx

```diff
@@ -0,0 +1,39 @@
+// Пример Dialog компонента
+import { CloseOutlined } from '@ant-design/icons'
+import { FocusTrap } from 'focus-trap-react'
+import styles from './styles.module.scss'
+import type { JSX } from 'react'
+
+interface DialogProps {
+  show: boolean
+  title: string | JSX.Element
+  onClose: () => void
+  children: React.ReactNode
+}
+
+export const Dialog = ({ show, title, children, onClose }: DialogProps) =>
+  show ? (
+    <div className={styles.overlay}>
```

Comment:
This ternary is too stretched. Personal preference, but for me it's more readable:

```suggestion
export const Dialog = ({ show, title, children, onClose }: DialogProps) => {
  if (!show) {
    return
  }
  
  return (
    <div className={styles.overlay}>
      <div aria-modal="true" className={styles.container} role="dialog">
```